js.nanorc

```diff
@@ -10,6 +10,13 @@ color yellow "\<[-+]?([1-9][0-9]*|0[0-7]*|0x[0-9a-fA-F]+)([uU][lL]?|[lL][uU]?)?\
 color yellow "\<[-+]?([0-9]+\.[0-9]*|[0-9]*\.[0-9]+)([EePp][+-]?[0-9]+)?[fFlL]?"
 color yellow "\<[-+]?([0-9]+[EePp][+-]?[0-9]+)[fFlL]?"
 
+## JSX tags
+color cyan   "<[^> ]+"
+color cyan   "<[a-zA-Z][a-zA-Z0-9]*>"
```

Comment:
Maybe list HTML tags there? To differentiate a color of standard tags from components?

Reply:
Having a different color for standard tags rather than components could be nice, but don't you think listing them is overkill? I would suggest it's sufficient to say that `<[a-z]+>` is an HTML tag and `<[A-Z][a-zA-Z0-9]*>` is a component.

(Also `<[a-zA-Z][a-zA-Z0-9]*>` will not match `<>`, it will match at least `<[a-zA-Z]>`

Reply:
So what if someone uses all caps when they do their HTML??

```html
<HTML>
<BODY>
I LIKE SCREAMING CAPS!!!
</BODY>
</HTML>
```

dumb i agree, but I've seen this in more than one place...

Reply:
About `*`/`+` you are right, it's my bad, sorry))

About overkill: listing of JS reserved words is not overkill...

HTML has 144 tags. Reference https://developer.mozilla.org/en-US/docs/Web/HTML/Element
```js
document.body.innerText.match(/<\w+>/g).reduce((acc, cur) => {
  if (!acc.includes(cur)) {
    acc.push(cur)
  }
  return acc
}, [])
```
Run this code in the console on a reference page.